strangerstudios/paid-memberships-pro
# Story 302 Customize section titles on account page

* Adds support to pass a title attribute to [pmpro_account] shortcode when it has a single section as param.

<img width="1220" alt="image" src="https://user-images.githubusercontent.com/1678457/235908972-c1606bda-d53e-4013-b348-a747bf7ff668.png">
<img width="904" alt="image" src="https://user-images.githubusercontent.com/1678457/235909093-a81e3a1c-e13b-4909-85ab-582557113ec4.png">

<img width="1259" alt="image" src="https://user-images.githubusercontent.com/1678457/235909303-b97021a3-cd93-4894-87e6-b8ad7ea9a4a2.png">


<img width="1578" alt="image" src="https://user-images.githubusercontent.com/1678457/235527950-a4ddbd4a-130d-4d82-aa2d-0958d74ceebd.png">
<img width="753" alt="image" src="https://user-images.githubusercontent.com/1678457/235527998-9b2cd8e6-55c9-4fc1-bdc5-f9240dc2e8f5.png">
<img width="1502" alt="image" src="https://user-images.githubusercontent.com/1678457/235528113-1e2760d0-7c52-4620-9611-5df37d70a2b8.png">
<img width="817" alt="image" src="https://user-images.githubusercontent.com/1678457/235528144-7ef375a2-521d-48ab-bd8b-9b88efb284c2.png">
<img width="1300" alt="image" src="https://user-images.githubusercontent.com/1678457/235528212-1a060975-6aca-49f1-ade6-b52bb72457b3.png">
<img width="758" alt="image" src="https://user-images.githubusercontent.com/1678457/235528246-2c264b8c-78d4-49ef-8b9b-d87d66a87252.png">


### All Submissions:






### Changes proposed in this Pull Request:

Extract from shortcode attributes the "title" and if it exists and meet conditions use it to render given single section. Otherwise ignore it and use default.



Resolves XXX.

### How to test the changes in this Pull Request:

1. Create a page or post
2. Add the shortcode and play with different combinations, one section, more than one, add title, make it empty, don't add it.
3. Check results.

### Other information:




### Changelog entry

> Enter a summary of all changes on this Pull Request. This will appear in the changelog if accepted.
 Adds support to pass a title attribute to [pmpro_account] shortcode when it has a single section as a param.
